docker-compose.yml

```diff
@@ -69,7 +69,7 @@ services:
 
   # Temporary: Remove when we've upgraded to ES6
   elasticsearch6:
-    image: docker.elastic.co/elasticsearch/elasticsearch:6.3.1
+    image: quay.io/centerforopenscience/elasticsearch/es6-arm-6.3.1
```

Comment:
Hardware specific overrides for ARM should go into `docker-compose-dist-arm64.override.yml`, not the base `docker-compose.yml`.

Just about everything else in that ARM `override` file could be removed, too, as it's no longer required since the Python upgrade to 3.12.

Should `README-docker-compose.md` also be updated accordingly?